include/dds/core/xtypes/detail/PrimitiveTypes.hpp

```diff
@@ -66,6 +66,8 @@ struct dynamic_type_traits<uint32_t>
     static constexpr const char* NAME = "uint32_t";
 };
 
+struct AAAAA : public dynamic_type_traits<uint32_t> {};
```

Comment:
Are you sure you want to leave the name like that?

Reply:
Upss, this line should be deleted